storehouse.py

```diff
@@ -98,6 +103,8 @@ def save_evc(self, evc):
 
     def _save_evc_callback(self, _event, data, error):
         """Display the save EVC result in the log."""
+        self._lock.release()
+        log.info(f'Lock {self._lock} acquired.')
```

Comment:
I think this log may be removed or changed to the debug level.
```suggestion
        log.info(f'Lock {self._lock} released.')
```

Reply:
Changes made.